internal/storetesting/search.go

```diff
@@ -0,0 +1,287 @@
+// Copyright 2014 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package storetesting
+
+import (
+	"sort"
+
+	charm "gopkg.in/juju/charm.v6"
+	"gopkg.in/juju/charmrepo.v3/csclient/params"
+
+	"gopkg.in/juju/charmstore.v5/internal/mongodoc"
+	"gopkg.in/juju/charmstore.v5/internal/router"
+	"gopkg.in/juju/charmstore.v5/internal/series"
+)
+
+// SearchSeries contains the list of charm series that will be indexed in
+// the search in descending order of preference.
+var SearchSeries = func() []string {
+	var ss []string
```

Comment:
Not sure it makes sense to execute this more than once.

Reply:
It is only executed once, at init time.